hasktorch/src/Torch/Typed/NN.hs

```diff
@@ -247,7 +244,7 @@ conv1d Conv1d {..} input = Torch.Typed.Native.conv1d @stride @padding
   (toDependent conv1dBias)
   input
 
-instance A.Parameterized (Conv1d inputChannelSize outputChannelSize kernelSize dtype device)
+-- instance A.Parameterized (Conv1d inputChannelSize outputChannelSize kernelSize dtype device)
```

Comment:
New typed Parameterized class does not need this instance?

Reply:
no, it's fully automatically derived for Models with a `Generic` instance.

Reply:
It is great.

Reply:
Do you have any reason to use comment-out?
I think we can delete it.

Reply:
actually, I decided to add these instances back.
I looked at the additional work necessary to port the examples to the new typed `Parameterized` class, and I think it warrants another PR.
so let's keep these instance around a little while longer.

Reply:
🆗